src/components/spinner/spinner.jsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import styles from './spinner.css';
+
+const SpinnerComponent = function (props) {
+    const {
+        className
+    } = props;
+    return (
+        <div
+            className={classNames(
+                styles.spinner,
+                className
```

Comment:
It might be simpler to use props.className instead of deconstructing the object here

Reply:
I didn't understand this before I merged, but @fsih explained it to me and now I do :)